app/Migrations/20220313170810_Initial.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace backend.Migrations
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
jeśli jest to wygenerowane jakimś toolem, to dodaj proszę w komentarzu

Reply:
jest to wygenerowane toolem

Reply:
Cały folder app/Migrations powstaje w wyniku wygenerowania przez Visual Studio migracji zmian w kodzie (w modelach) do layoutu bazy. Chyba nie powinniśmy modyfikować tych plików.